src/frontend/src/flows/about/index.ts

```diff
@@ -97,7 +97,7 @@ export const aboutView = (): void => {
   document.title = "About | Internet Identity";
   const container = document.getElementById("pageContent") as HTMLElement;
   const i18n = new I18n();
-  if (process.env.HYDRATE_STATIC_PAGES !== "0") {
+  if (process.env.II_HYDRATE_STATIC_PAGES !== "0") {
```

Comment:
out of curiosity: why the rename? Other options are actual II (runtime) option, hence why they're prefixed with `II_`. This is a build option that could do without the prefix (for clarity)

Reply:
> Only VITE_SOME_KEY will be exposed as import.meta.env.VITE_SOME_KEY to your client source code, but DB_PASSWORD will not.
>
> console.log(import.meta.env.VITE_SOME_KEY) // 123
> console.log(import.meta.env.DB_PASSWORD) // undefined

https://vitejs.dev/guide/env-and-mode.html

We use `II_` instead of `VITE_`. So when we will replace `process.env` it might become `undefined`.

Reply:
`.env` removed.